_posts/2020-03-30-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Steering Group executive summary, action items, and minutes
+author: Franklin Feingold
+display: true
+---
+
+# Steering Group executive summary, action items, and minutes
+
+Date: Monday March 23, 2020
+
+<!--more-->
+
+### Executive Summary
+
+We discussed how to move forward with implementing the author lists on [Zenodo](https://zenodo.org/record/3686062){:target="_blank"} from our pdf specification. We want to provide guidance on how we landed on our decision: a single author “BIDS Contributing Group”. This will be opened as a proposal with a request for comments (RFC) on the [BIDS-Specification repository](https://github.com/bids-standard/bids-specification){:target="_blank"}. This is important to deploy with the community’s approval. This may give way for a stronger contributor tracking system.
+
+We clarified the format and points for the monthly Maintainers update. This should be less than 1 page and keep a high level view of BIDS. The executive summary must include: achievements, struggles, next month aims, blockers or steering group help. The update may also link to issues with a 1 sentence narrative. Keeping in mind the purpose of the update from the Steering Group’s perspective: knowing the hard parts or blockers and capture the work needed to maintain BIDS.
+
+We invited the [ASL BEP](https://docs.google.com/document/d/15tnn5F10KpgHypaQJNNGiNKsni9035GtDqJzWqkkP6c/edit#heading=h.prrzvwchfio6){:target="_blank"} leads to the next Steering Group meeting.
```

Comment:
I recommend to use the following URL style for BEPs:
```suggestion
We invited the [ASL BEP](https://bids.neuroimaging.io/bep005){:target="_blank"} leads to the next Steering Group meeting.
```

Reply:
try it, it works :) --> https://bids.neuroimaging.io/bep005

the benefit is, that it'll be more stable, because we can manage where it is pointing to from https://github.com/bids-standard/bids-website/tree/gh-pages/_bep_collection

Reply:
yep just tested all looks good!